File: src/Entités/ReservationM.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entités;

/**
 *
 * @author CHEDI
 */
public class ReservationM {
    private int id_res,nb_chambre,nb_personne,id_maison;
    private String date;
    
      public ReservationM(int nb_chambre, int nb_personne , String date,int id_maison) {
        this.nb_chambre = nb_chambre;
        this.nb_personne = nb_personne;
        this.date = date;
        this.id_maison=id_maison;
    }

    public ReservationM(int id_res, int nb_chambre, int nb_personne , String date,int id_maison) {
        this.id_res = id_res;
        this.nb_chambre = nb_chambre;
        this.nb_personne = nb_personne;
        this.date = date;
        this.id_maison=id_maison;
    }

    public int getId_res() {
        return id_res;
    }

    public int getNb_chambre() {
        return nb_chambre;
    }

    public int getNb_personne() {
        return nb_personne;
    }

    public String getDate() {
        return date;
    }
    
    public int getId_maison(){
        return id_maison;
    }

    public void setId_res(int id_res) {
        this.id_res = id_res;
    }

    public void setNb_chambre(int nb_chambre) {
        this.nb_chambre = nb_chambre;
    }

    public void setNb_personne(int nb_personne) {
        this.nb_personne = nb_personne;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public void setId(int id_maison){
        this.id_maison=id_maison;
    }

    public void setId_maison(int id_maison) {
        this.id_maison = id_maison;
    }

    @Override
    public String toString() {
        return "ReservationM{" + "id_res=" + id_res + ", nb_chambre=" + nb_chambre + ", nb_personne=" + nb_personne + ", date=" + date + ", id_maison=" + id_maison + '}';
    }
    

    

   

  
    
}

File: Desktop/gestionevenement/src/Entity/Reservationm.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reservationm
 *
 * @ORM\Table(name="reservationm", indexes={@ORM\Index(name="maisonH_reservationm", columns={"id_maison"})})
 * @ORM\Entity
 */
class Reservationm
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_res", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idRes;

    /**
     * @var int
     *
     * @ORM\Column(name="nb_chambre", type="integer", nullable=false)
     */
    private $nbChambre;

    /**
     * @var int
     *
     * @ORM\Column(name="nb_personne", type="integer", nullable=false)
     */
    private $nbPersonne;

    /**
     * @var string
     *
     * @ORM\Column(name="date", type="string", length=50, nullable=false)
     */
    private $date;

    /**
     * @var \Maisonh
     *
     * @ORM\ManyToOne(targetEntity="Maisonh")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_maison", referencedColumnName="id_maison")
     * })
     */
    private $idMaison;

    public function getIdRes(): ?int
    {
        return $this->idRes;
    }

    public function getNbChambre(): ?int
    {
        return $this->nbChambre;
    }

    public function setNbChambre(int $nbChambre): self
    {
        $this->nbChambre = $nbChambre;

        return $this;
    }

    public function getNbPersonne(): ?int
    {
        return $this->nbPersonne;
    }

    public function setNbPersonne(int $nbPersonne): self
    {
        $this->nbPersonne = $nbPersonne;

        return $this;
    }

    public function getDate(): ?string
    {
        return $this->date;
    }

    public function setDate(string $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getIdMaison(): ?Maisonh
    {
        return $this->idMaison;
    }

    public function setIdMaison(?Maisonh $idMaison): self
    {
        $this->idMaison = $idMaison;

        return $this;
    }


}
